### ISSUE TITLE
Error when receiving picture

### ISSUE THREAD

**Issue by GuangChen2333** [TWT233/khl.py#11]
An exception occurred when the robot received a picture.

### Error message: 
```
Task exception was never retrieved
future: <Task finished name='Task-33' coro=<Bot._event_handler.<locals>._dispatch_msg() done, defined at D:\Python\Python39\lib\site-packages\khl\bot.py:104> exception=AttributeError("'NoneType' object has no attribute 'type'")>
Traceback (most recent call last):
  File "D:\Python\Python39\lib\site-packages\khl\bot.py", line 107, in _dispatch_msg
    if m.type == Msg.Types.SYS:
AttributeError: 'NoneType' object has no attribute 'type'
```

### How to reproduce: 
Just send a picture

### Operating environment:
OS: Windows 10.0.19043.1151
Python v3.9.5:0a7dcbd
khl.py 0.0.9

**Comment by TWT233** [comment]
thx for rep, fix scheduled done in 2days

**Comment by TWT233** [comment]
already done in eb306de5f271f272a7f80c87b7688d0fbf793f95, the commit is merged into master and released with v0.0.10, try update your khl.py
